docs/containers-storage.conf.5.md

```diff
@@ -53,12 +53,48 @@ The `storage.options` table supports the following options:
   Maximum size of a container image.   This flag can be used to set quota on the size of container images. (default: 10GB)
 
 **mount_program**=""
-  Specifies the path to a custom program to use instead for mounting the file system.
+  Specifies the path to a custom program to use instead of using kernel defaults for mounting the file system.
+
+      mount_program = "/usr/bin/fuse-overlayfs"
 
 **mountopt**=""
 
   Comma separated list of default options to be used to mount container images.  Suggested value "nodev".
 
+**remap-uids=**""
+**remap-gids=**""
+
+  Remap-UIDs/GIDs is the mapping from UIDs/GIDs as they should appear inside of
+a container, to UIDs/GIDs as they should appear outside of the container, and
```

Comment:
suggest "to UIDs/GIDs" to "the UIDs/GIDs"

Reply:
"outside of the container" is throwing me, but I don't have a good suggestion for better wording...

Reply:
Fixed